config/jest.config.js

```diff
@@ -18,6 +18,11 @@ module.exports = {
   unmockedModulePathPatterns: ['react', 'node_modules/'],
   moduleNameMapper: {
     '^.+\\.(png|gif|css|scss)$': 'identity-obj-proxy',
+    "^dnd-core$": "dnd-core/dist/cjs/index.js",
+    "^react-dnd$": "react-dnd/dist/cjs/index.js",
+    "^react-dnd-test-backend$": "react-dnd-test-backend/dist/cjs/index.js",
+    "^react-dnd-test-backend/TestBackend$": "react-dnd-test-backend/dist/cjs/TestBackend.js",
+    "^react-dnd-test-utils$": "react-dnd-test-utils/dist/cjs/index.js"
```

Comment:
Theese are fixing problems with jest and modules. `react-dnd` doesn't use common js implementation by default [1].

[1] http://react-dnd.github.io/react-dnd/docs/testing#setup